runtime/protocol/http-client/common/test/aws/smithy/kotlin/runtime/http/engine/EnvironmentProxySelectorTest.kt

```diff
@@ -49,6 +49,10 @@ class EnvironmentProxySelectorTest {
     private val tests = listOf(
         // no props
```

Comment:
For completeness should we also test a blank string for the `http.nonProxyHosts` property and `NO_PROXY` environment variable?